src/App.tsx

```diff
@@ -1,24 +1,283 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import cn from 'classnames';
+
+import * as todoService from './api/todos';
+import { Status } from './types/Status';
+import { Todo } from './types/Todo';
+
 import { UserWarning } from './UserWarning';
+import { ErrorMessage } from './types/ErrorMessage';
 
-const USER_ID = 0;
+const USER_ID = 11074;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<ErrorMessage | null>(null);
+  const [filterStatus, setFilterStatus] = useState(Status.ALL);
+  const [todoTitle, setTodoTitle] = useState<string>('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
+    todoService.getTodos(USER_ID)
+      .then((todosFromServer) => {
+        setTodos(todosFromServer);
+      })
+      .catch(() => {
+        setErrorMessage(ErrorMessage.LOAD);
+      });
+  }, []);
+
+  useEffect((): (() => void) | undefined => {
```

Comment:
The syntax is incorrect 
```suggestion
  useEffect(()=> {
```